core/embed/rust/src/ui/display.rs

```diff
@@ -1,7 +1,9 @@
 use crate::{micropython::time, time::Duration, trezorhal::display};
 
-#[cfg(not(feature = "model_tt"))]
+#[cfg(feature = "model_t1")]
 use crate::ui::model_t1::constant;
+#[cfg(feature = "model_tr")]
+use crate::ui::model_tr::constant;
 #[cfg(feature = "model_tt")]
 use crate::ui::model_tt::constant;
```

Comment:
@grdddj does vscode actually break on this? For me it highlighted these lines as incorrect but otherwise the analyzer features worked fine. I'm OK with that result. I would go for explicit mutual exclusion only in case this breaks in a more serious way (e.g. impossible to follow definitions)

Reply:
It does not break anything, it just complains - no real error (not even warning) during compilation (which is completely outside of VSCode).